lib/ace/http/service.ex

```diff
@@ -59,7 +59,11 @@ defmodule Ace.HTTP.Service do
     quote do
       def start_link(initial_state, options \\ []) do
         # DEBT Remove this for 1.0 release
-        behaviours = __MODULE__.module_info() |> get_in([:attributes, :behaviour])
+        behaviours =
+          __MODULE__.module_info()
```

Comment:
I wonder if it is worth using something from `Raxx` to do this check. I think there is a `Raxx.check_behaviour`